src/lib/model/OutboundTransfersModel.js

```diff
@@ -317,6 +317,9 @@ class OutboundTransfersModel {
             }, ASYNC_TIMEOUT_MILLS);
 
             // create a quote request
+            const quote = this._buildQuoteRequest();
+            this.data.quoteId = quote.quoteId;
+
```

Comment:
this quote is created 12 lines above this, any reason why it is duplicated and moved here?